modules/cudl-data-processing/provider.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    aws = {
+      source                = "hashicorp/aws"
+      configuration_aliases = [aws.us-east-1]
+    }
+  }
+}
```

Comment:
By convention the `terraform` block should go in a file called `terraform.tf`. I see this fixes the warning about the undefined provider which makes sense. Also this should probably have an extra newline on the end to fix GitHub's "please add a newline" warning symbol

Reply:
Updated to terraform.tf and added extra line.